common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -124,24 +126,25 @@ const LiveBlockContainer = ({
 			 * - 'pending' is used to mark blocks that have been inserted as part of a live update. We use this
 			 *    to animate the reveal as well as for enhancing twitter embeds
 			 */
-			className={`block ${isLiveUpdate && "pending"}`}
+			className={`block ${isLiveUpdate && 'pending'}`}
 			css={css`
 				padding: ${space[2]}px ${SIDE_MARGIN_MOBILE}px;
 				margin-bottom: ${space[3]}px;
 				background: ${neutral[100]};
-				border-top: 1px solid ${borderColour};
+				border-top: ${isPinnedPost ? 0 : `1px solid ${borderColour}`};
+				border-bottom: ${isPinnedPost ? 0 : `1px solid ${neutral[86]}`};
 				${from.tablet} {
 					padding: ${space[2]}px ${SIDE_MARGIN}px;
 					padding-left: ${LEFT_MARGIN_DESKTOP}px;
 				}
-				border-bottom: 1px solid ${neutral[86]};
 			`}
 		>
 			<Header>
 				{blockFirstPublished && (
 					<FirstPublished
 						firstPublished={blockFirstPublished}
 						blockLink={blockLink}
+						isPinnedPost={isPinnedPost ? false : true}
```

Comment:
I think this is reversing the logic here. Does it work if we do this?
```suggestion
						isPinnedPost={isPinnedPost}
```